src/library/features/crates/cratefeaturehelper.cpp

```diff
@@ -74,6 +74,68 @@ CrateId CrateFeatureHelper::createEmptyCrate() {
     if (m_pTrackCollection->insertCrate(newCrate, &newCrateId)) {
         DEBUG_ASSERT(newCrateId.isValid());
         newCrate.setId(newCrateId);
+        m_pTrackCollection->crates().initClosureForCrate(newCrateId);
+        m_pTrackCollection->crates().generateCratePaths(newCrate);
+        qDebug() << "Created new crate" << newCrate;
+    } else {
```

Comment:
CrateFeatureHelper::createEmptyCrate only makes top-level crates, right? If so, it should be renamed to clarify that.